docs/architecture/gameStateUtil.md

```diff
@@ -0,0 +1,53 @@
+# GameStateUtil
+
+The [GameStateUtil](../src/utils/gameStateUtil.ts) enforces a valid [GameState](./GameState)
```

Comment:
should be `../../src/utils/gameStateUtil.ts`

Reply:
Fixed